app/cdap/components/GridTable/index.tsx

```diff
@@ -304,17 +319,11 @@ export default function GridTable() {
       <ToolBarList
         setShowBreadCrumb={setShowBreadCrumb}
         showBreadCrumb={showBreadCrumb}
-        columnType={columnType}
+        columnType={columnType} // TODO: column type needs to be send dynamically after integrating with transfomations branch
         submitMenuOption={(option, datatype) => {
-          setSnackbar(() => ({
-            open: true,
-            isSuccess: true,
-            message: 'Function Selected',
-          }));
-          return false;
-          // TODO: will integrate with add transformation panel later
+          !transformationOptions.includes(option) ? onMenuOptionSelection(option, datatype) : null;
         }}
-        disableToolbarIcon={gridData?.headers?.length > 0 ? false : true}
+        disableToolbarIcon={gridData?.headers?.length === 0 ? true : false}
```

Comment:
`gridData?.headers?.length === 0` is fine here - this doesn't need a ternary.

Reply:
disableToolbarIcon type is assigned as boolean we will be disabling toolbar if no data on grid is present
Rewriting above ternary expression as !Boolean(gridData?.headers?.length)